doc/Language/grammar_tutorial.pod6

```diff
@@ -27,41 +27,75 @@ language.
 
 =SUBTITLE The Broad Concept of Perl Grammars
 
-Regular Expressions (regex) work well to find patterns in strings and manipulate them. However, when you need to find multiple patterns at once, or need to combine patterns, or test for patterns that may surround strings, or other patterns - regular expressions alone are not adequate.
```

Comment:
My thoughts are: you probably don't want to consistently re-wrap lines as you make edits. Messes with diff, doesn't it? I had an issue earlier with this. if you re-wrap lines, you have to make sure you don't do it to embedded code examples (I'm guessing) since you'll need to deal with comments and strings and other stuff that will break if you just move the hanging bits to the next line. I'm new to editing POD, so there may be already available solutions to this.